### PR TITLE
This is to add the skeleton of translatable glossary and the fix of zh-cn for issue#1224.

### PR DESCRIPTION
**Pull request by dongxuwang**
You're welcome to try this pr locally, in condition that you have to remove the _site directory at first(which should be regenerated  by jeklly.)

### CODE REVIEW

File: _zh-cn/glossary/index.md [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+在实参上 _调用_ 方法、函数或闭包，意即其方法体会在指定实参上执行。
+
+* #### Java虚拟机（JVM）
+_JVM_ 是Java虚拟机(#runtime)的缩写，或叫[运行时](#runtime)，是运行Scala程序的宿主。
+
+* #### 字面量（literal）
+`1`，`"One"`，和`(x: Int) => x + 1`是 _字面量_ 的例子，字面量是描述对象的便捷方式，便捷在这种方式正好反映了所创建对象的结构。
+
+* #### 本地函数（local function）
+_本地函数_ 是块内`def`定义的，作为对比，同为`def`定义的作为类、特质或单例对象的成员则被称作[方法](#method)。
```
Comment (liufengyun) [inline _zh-cn/glossary/index.md]: `本地函数` -> `局部函数`？

File: _zh-cn/glossary/index.md [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+
+* #### 本地变量（local variable）
+_本地变量_ 是块内`val`或`var`定义的。尽管函数参数和[本地变量](#local-variable)类似，但并不叫本地变量，而是去掉"本地"直接叫"参数"或"变量"。
```
Comment (liufengyun) [inline _zh-cn/glossary/index.md]: `局部变量`?

**Comment by dongxuwang** [comment]
听上去更文档化一些，多谢

File: _zh-cn/glossary/index.md [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+带有类型参数的特质。例如，因`scala.collection.Set`带一类型参数，故其为泛型特质。
+
+* #### 守卫（guard）
+参见[过滤器](#filter).
+
+* #### 助手函数（helper function）
+目的是为一个或多个其他邻近函数提供服务的函数。助手函数常实现为本地函数。
+
+* #### 助手方法（helper method）
+作为类成员的[助手函数](#helper-function)。助手方法常为私有方法。
```
[the remark this answers is not in the document]
Comment (dongxuwang) [reply _zh-cn/glossary/index.md]: 👍

File: _zh-cn/glossary/index.md (hunk 1 above)
Comment (dongxuwang) [reply _zh-cn/glossary/index.md]: 👍

File: _zh-cn/glossary/index.md (hunk 2 above)
Comment (dongxuwang) [reply _zh-cn/glossary/index.md]: 👍

File: _glossary/index.md [hunk 4]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: glossary
 title: Glossary
+
+languages: [zh-cn]
```
Comment (liufengyun) [inline _glossary/index.md]: I'm not sure about this setting -- I'm not sure what's the convention for scala-lang.org, but it seems it's possible to make the data available for the whole site by using _yaml data files_.

Comment (dongxuwang) [reply _glossary/index.md]: The is supposed to be the correct way. See [cheatsheet](https://docs.scala-lang.org/cheatsheets/index.html) is doing the same thing in its [source code](https://github.com/scala/docs.scala-lang/edit/master/_overviews/cheatsheets/index.md).

**Comment by SethTisue** [comment]
@liufengyun should we merge?

**Pull request merged**